Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
+COPY . .
+CMD npm run start
+EXPOSE 8182
```

Comment:
I would write this line before previous

Reply:
[Dockerization example](https://nodejs.org/ru/docs/guides/nodejs-docker-webapp/)

Reply:
Fixed. But the order of this config is not mandatory.

From docks:
`The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports.`
https://docs.docker.com/engine/reference/builder/#expose